config/routes.rb

```diff
@@ -13,8 +13,10 @@
 
   get '/my_pokemons', to: 'pokemons#my_pokemons'
 
-  resources :bookings, only: [:index, :show, :edit, :update]
+  resources :bookings, only: [:index, :show, :edit, :update, :destroy]
```

Comment:
Do we need to destroy a booking? I would think that the status is enough to manage it's life cycle no?